tests/featureTesting/compare_csv.py

```diff
@@ -140,26 +104,41 @@ def remove_outliers_using_score(self, pandas_df):
             key_1 = (mz_1, rt_1)
             key_2 = (mz_2, rt_2)
 
-            if key_1 in mzrt1 or key_2 in mzrt2:
+            if key_1 in mzrt1 and key_2 in mzrt2:
                 pandas_df.drop(index, inplace=True)
             else:
                 mzrt1.append(key_1)
                 mzrt2.append(key_2)
 
         return pandas_df
 
-    def plot(self, pandas_df, sample_list):
+    def remove_outliers(self, pandas_df):
+        """
+        Remove rows whose mzs and rts are not equal
+        Args:
+            pandas_df (df): Merged pandas dataframe
```

Comment:
@DeepakSharmaElucidata Try using semantic_names of the data frames which can be easily understood by their names because these are all pandas data frame like you mentioned if it is a merged data frame, the name `merged_dataframe` would make more sense here.

Reply:
Done